locales-src/asrouter.ftl

```diff
@@ -90,17 +90,42 @@ cfr-whatsnew-button =
   .label = What’s New
   .tooltiptext = What’s New
 
+cfr-whatsnew-fx70-hero-title = { -brand-short-name } now fights harder for your privacy
+cfr-whatsnew-fx70-hero-body = The latest update enhances the Tracking Protection feature and makes it easier than ever to create secure passwords for every site.
+
+cfr-whatsnew-fx70-notes-body = Protection reports, Secure Proxy for public Wifi, improved account controls, and many more safety and security tools are included in the latest update.
+cfr-whatsnew-fx70-notes-link-text = Read the release notes
```

Comment:
Not all locales have release notes… I suppose we should make sure to remotely target this panel message to the appropriate locales. @piatra can we optionally not have a link and just have the body as a release-notes 1-liner?

Reply:
All locales but en-US don't have release notes ;-)